README.md

```diff
@@ -35,23 +38,28 @@ and uv
 pip install uv
 uv tool install promdens
 ```
-
-
-### Usage
-The code requires information about the method (PDA or PDAW), the number of excited states to consider, 
-the number of initials conditions to be generated, and the characteristics of the laser pulse, such as the envelope type 
-(Gaussian, Lorentzian, sech, etc.), the pulse frequency, the linear chirp parameter, and the full width at half maximum parameter. 
-The code can be launched from a terminal with a series of flags as follows
-
-```console
-promdens --method pda --energy_units a.u. --tdm_units debye --nstates 2 --fwhm 3 --omega 0.355 --npsamples 10 --envelope_type gauss input_file.dat
-```
-
-The input file should contain information about the excitation energies and magnitudes of the transition dipole moments 
-for each pair of sampled nuclear positions and momenta (labelled by an index number).
+## Brief outline of the code 
+The code makes several steps and analysis procedures before it calculates the promoted density quantities.
+The **PROMDENS** workflow is briefly summarized as:
+1) The absorption spectrum is calculated using the Nuclear Ensemble Approach and decomposed into different excited states. 
+2) The laser field and its characteristics such as spectral intensity are calculated. 
+3) The code checks if the pulse fulfils Maxwell equations, i.e., is not too short.
```

Comment:
I wanted to show the structure of the code. It might be useful to tell people they will also get a spectrum from it and an analysis of the pulse. I will think about some improvements for the future.

Reply:
Aah, I see. I wanted to emphasise it since it's quite a cool feature (at least for me). I will keep it for now but think about it for the next modifications. But thanks for all the comments!!